common/prque/prque_byteSlice.go

```diff
@@ -0,0 +1,65 @@
+// CookieJar - A contestant's algorithm toolbox
```

Comment:
@winnie-byun Can we use golang native heap instead of 3rd party pkg?
http://golang.site/go/article/208-Go%EC%97%90%EC%84%9C-Heap-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0